<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartProgram</class>
 <widget class="QWidget" name="StartProgram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>574</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>756</width>
    <height>574</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Start Program</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 108);</string>
  </property>
  <widget class="QPushButton" name="okProcessCount">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#okProcessCount:hover{
	background-color: rgb(209, 69, 0);
};
background-color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="lableProcessCount">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboProcessCount">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>One</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Two</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Three</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Four</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Five</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Six</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Seven</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Eight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ten</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
